packages/reason-generate-types-from-graphql-schema/reason-generate-types-from-graphql-schema.0.7/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+maintainer: "greg <greg@hackages.io>"
+authors: "greg <greg@hackages.io>"
+build: [
+  ["ocamlbuild -r -use-ocamlfind  src/index.native"]
+]
+tags: ["reasonml" "graphql"]
+dev-repo: "https://github.com/Gregoirevda/reason-generate-types-from-graphql-schema.git"
+bug-reports: "https://github.com/Gregoirevda/reason-generate-types-from-graphql-schema/issues"
+homepage: "https://github.com/Gregoirevda/reason-generate-types-from-graphql-schema"
+install: ["./index.native"]
+remove: ["ocamlfind" "remove" "reason-generate-types-from-graphql-schema"]
```

Comment:
you are not installing anything (see previous point), so you don't need to uninstall the library.